src/Uno.Extensions.Maui.WinUI.Markup/Uno.Extensions.Maui.WinUI.Markup.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="..\tfms-ui-maui.props" />
+
+	<PropertyGroup>
+		<UseMaui>true</UseMaui>
+		<RootNamespace>Uno.Extensions.Maui</RootNamespace>
+		<NoWarn>$(NoWarn);NU5104;NU5048;NU1009</NoWarn>
+		<Description>Extensions to embed .NET MAUI controls within your Uno app using C# Markup.</Description>
+		<!--Temporary disable missing XML doc until fixed in the whole package-->
+		<WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
+
+		<IsMauiEmbedding>false</IsMauiEmbedding>
```

Comment:
We have this logic for setting IsMauiEmbedding in I think three places now - should we just import a props file?